app/src/main/res/values/strings.xml

```diff
@@ -92,4 +92,11 @@
     <string name="LovingLife">What I am Loving about Life right now:</string>
     <string name="social_media">Social Media</string>
     <string name="users_profile_picture">Users Profile Picture</string>
+    <string name="weight">Weight:</string>
+    <string name="_0">0</string>
+    <string name="pounds_lbs">Pounds (lbs)</string>
+    <string name="height">Height:</string>
+    <string name="inches">Inches</string>
+    <string name="calculate">Calculate</string>
+    <string name="normal_range">(Normal Range is 18.5 - 24.9)</string>
```

Comment:
Something to note moving forward is that if we are going to implement "Themes" in the settings (night/light/other), we want to keep the `night/strings.xml` file up-to-date with this file. Not sure about on your machine, but mine spits out warnings/errors when the two files don't match.

Reply:
@jacobburgo I will go ahead and add all my strings to the night/strings.xml file for my next Pull Request.